tests/acceptance/cli/test_status.py

```diff
@@ -1,115 +0,0 @@
-
-
-
-import csv
-import re
-
-import pytest
-
-from nucypher.blockchain.eth.agents import (
-    AdjudicatorAgent,
-    ContractAgency,
-    NucypherTokenAgent,
-)
-from nucypher.blockchain.eth.signers.software import Web3Signer
-from nucypher.cli.commands.status import status
-from nucypher.config.constants import TEMPORARY_DOMAIN
-from nucypher.crypto.powers import TransactingPower
-
-
-@pytest.mark.skip()
-def test_nucypher_status_network(click_runner, testerchain, agency_local_registry):
-
-    network_command = ('network',
-                       '--registry-filepath', str(agency_local_registry.filepath.absolute()),
-                       '--eth-provider', TEST_ETH_PROVIDER_URI,
-                       '--network', TEMPORARY_DOMAIN)
-
-    result = click_runner.invoke(status, network_command, catch_exceptions=False)
-    assert result.exit_code == 0
-
-    token_agent = ContractAgency.get_agent(NucypherTokenAgent, registry=agency_local_registry)
-    staking_agent = ContractAgency.get_agent(StakingEscrowAgent, registry=agency_local_registry)
-    adjudicator_agent = ContractAgency.get_agent(AdjudicatorAgent, registry=agency_local_registry)
-
-    agents = (token_agent, staking_agent, adjudicator_agent)
-    for agent in agents:
-        contract_regex = f"^{agent.contract_name} \\.+ {agent.contract_address}"
-        assert re.search(contract_regex, result.output, re.MULTILINE)
-
-    assert re.search(f"^Provider URI \\.+ {TEST_ETH_PROVIDER_URI}", result.output, re.MULTILINE)
-    assert re.search(f"^Current Period \\.+ {staking_agent.get_current_period()}", result.output, re.MULTILINE)
-
-
-@pytest.mark.skip()
-def test_nucypher_status_events(click_runner, testerchain, agency_local_registry, staking_providers, temp_dir_path):
```

Comment:
This test might still be applicable...

Reply:
Is this command being used?  This test has been skipped for quite some time and there are name errors inside.  Admittedly this removal is a little bit out of scope.

Reply:
Ah right, I missed that the test was skipped. I see now. The issue is that the status cli retrieves legacy StakingEscrow and PolicyManager events (by using old versions of the contracts). We don't implicitly generate those events anymore as part of the testing suite, so testing was skipped.

It's probably fair to remove the tests - I don't see a simple way to get them working. Not in this PR, but we might need to think about just removing the `nucypher status events` functionality. It's only been kept for legacy reasons, which become less needed/applicable over time.